vsts/node-nightly-df.yaml

```diff
@@ -41,8 +39,12 @@ phases:
       IOT_PROVISIONING_ROOT_CERT: $(LINUX-IOT-PROVISIONING-ROOT-CERT)
       IOT_PROVISIONING_ROOT_CERT_KEY: $(LINUX-IOT-PROVISIONING-ROOT-CERT-KEY)
       IOT_PROVISIONING_SERVICE_CONNECTION_STRING: $(LINUX-IOT-PROVISIONING-SERVICE-CONNECTION-STRING)
+      DPS_CONN_STRING_INVALID_CERT: $(DPS-CONN-STRING-INVALID-CERT)
+      DPS_GLOBAL_DEVICE_ENDPOINT_INVALID_CERT: $(DPS-GLOBAL-DEVICE-ENDPOINT-INVALID-CERT)
+      IOTHUB_CONN_STRING_INVALID_CERT: $(IOTHUB-CONN-STRING-INVALID-CERT)
+      IOTHUB_DEVICE_CONN_STRING_INVALID_CERT: $(IOTHUB-DEVICE-CONN-STRING-INVALID-CERT)
 
-  - task: PublishTestResults@1
+  - task: PublishTestResults@2
```

Comment:
v1 used to work fine, and v2 had problems with glob patterns.. what changed that it doesn't work anymore?

Reply:
V1 was getting a path error.  Saying it was too long.  This was bogus.  The xml file paths and names  were no longer than before.  What had changed was that there was one additional xml from the server_validation test?  Going to v2 fixed it for me.

Reply:
This path issue had been noted as an issue for v1.